topic_dynamics/parsing.py

```diff
@@ -327,7 +344,7 @@ def uci_format(tokens_file: str, output_dir: str, name: str) -> None:
                 file_tokens_separated_numbered.append([sorted_dictionary_of_tokens[entry[0]], int(entry[1])])
             file_tokens_separated_numbered = sorted(file_tokens_separated_numbered, key=itemgetter(0), reverse=False)
             for entry in file_tokens_separated_numbered:
-                fout.write(str(line.split(";")[0]) + " " + str(entry[0]) + " " + str(entry[1]) + "\n")
+                fout.write(" ".join([str(line.split(";")[0]), str(entry[0]), str(entry[1])]) + "\n")
```

Comment:
use string formatting as suggested above - so it will be clear what fields you are writing